config/example/config.toml

```
@@ -676,6 +676,7 @@ active_space_ruins = [
 	"_maps/map_files220/RandomRuins/SpaceRuins/convoy_ambush.dmm",
 	"_maps/map_files220/RandomRuins/SpaceRuins/voxraiders_1.dmm",
 	"_maps/map_files220/RandomRuins/SpaceRuins/spacehotel.dmm",
+ 	"_maps/map_files220/RandomRuins/SpaceRuins/ABOTCH_RSN.dmm",
```

Название карты в snake case. Файл тоже переименовать
```suggestion
 	"_maps/map_files220/RandomRuins/SpaceRuins/abotch_rsn.dmm",
```